eos/nonleptonic-amplitudes/qcdf-amplitudes.cc

```diff
@@ -202,11 +211,11 @@ namespace eos
                 {
                     for (unsigned l = 0; l < 3; l++)
                     {
-                        A_alpha_qcdf += B[i] * p1[i][j] * C_u[0][j][k] * p2[k][l] * Lambda_u[l];
-                        A_alpha_qcdf += B[i] * p1[i][j] * Lambda_u[j] * C_u[1][l][k] * p2[k][l];
+                        A_alpha_qcdf += B[i] * p1[i][j] * C_u[0][j][k] * p2[k][l] * Lambda_u()[l];
```

Comment:
You could move the function call to ``Lambda_u`` outside of the for loop. This should speed things up a bit.